tests/rpc/calls/getTransactionReceipt.test.ts

```diff
@@ -0,0 +1,82 @@
+import { getTransactionReceiptHandler } from '../../../src/rpc/calls/getTransactionReceipt'
+import { RPCResponse } from '../../../src/types/types'
+
+describe('Test get transaction receipt request testnet', () => {
+  it('Returns transaction receipt', async () => {
+    const request = {
+      jsonrpc: '2.0',
+      method: 'eth_getTransactionReceipt',
+      params: [
+        '0x054b60a3d157a9b9a7d1734aaaf06db939ff5ae6b9c75e221cf8504f464ec0ef',
+      ],
+      id: 1,
+    }
+    const starkResult: RPCResponse = <RPCResponse>(
+      await getTransactionReceiptHandler(request)
+    )
+
+    expect(typeof starkResult.result).toBe('object')
+    expect(starkResult.result).toMatchObject({
+      transactionHash:
+        '0x54b60a3d157a9b9a7d1734aaaf06db939ff5ae6b9c75e221cf8504f464ec0ef',
```

Comment:
Will eth add zeroes at the beginning of hash? 

`0x054b60a3d157a9b9a7d1734aaaf06db939ff5ae6b9c75e221cf8504f464ec0ef` !== '0x54b60a3d157a9b9a7d1734aaaf06db939ff5ae6b9c75e221cf8504f464ec0ef'

Reply:
It seems like tools like Starkscan add an extra zero at the beginning of transaction hashes while Starknet RPC providers respond with transaction hashes without a leading zero even though they accept hashes with a leading zero. 
I've updated the logic to return hashes with a leading zero to match Ethereum's transaction hash length.
I've also updated the test.